### PR TITLE
feat: indexer

### PR DESCRIPTION
**Pull request by nejcm**
This implements indexer queries and hooks for nabla

### CODE REVIEW

File: src/components/Asset/Selector/Modal/index.tsx [hunk 1]
```diff
@@ -35,13 +35,13 @@ const AssetList = ({ assets, onSelect, selected }: AssetListProps): JSX.Element
...
             variant="secondary"
-            key={token.address}
+            key={token.id}
             onClick={() => onSelect(token)}
             className="w-full items-center justify-start gap-4 px-3 py-1 border-0 bg-[rgba(0,0,0,.2)] text-left hover:opacity-80 dark:bg-[rgba(255,255,255,.12)]"
           >
             <span className="relative">
-              <Avatar size="xs" letters={token.symbol} src={token.logoURI} shape="circle" className="text-xs" />
-              {selected == token.address && (
+              <Avatar size="xs" letters={token.symbol} /* src={token.logoURI} */ shape="circle" className="text-xs" />
```
Comment (ebma) [inline src/components/Asset/Selector/Modal/index.tsx:43]: What are we going to do about the token logo?

File: src/components/Pools/Swap/columns.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ export const coverageColumn: ColumnDef<SwapPoolColumn> = {
 export const aprColumn: ColumnDef<SwapPoolColumn> = {
   header: 'APR',
   accessorKey: 'apr',
-  accessorFn: (row) => row.apr?.toString(),
+  accessorFn: (row) => row.reserves?.toString(), // TODO: get apr
```
Comment (ebma) [inline src/components/Pools/Swap/columns.tsx:51]: Is this something that we calculate or that the indexer needs to provide?

File: src/components/Swap/Approval/index.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ export interface ApprovalProps {
 }
 
 const ApprovalSubmit = ({ token }: ApprovalProps): JSX.Element | null => {
-  const { router } = useGetTenantData(nablaConfig) || {};
+  const { router } = useGetTenantData(nablaConfig)[0] || {};
```
Comment (ebma) [inline src/components/Swap/Approval/index.tsx:13]: Can we maybe make this more expressive?

Comment (ebma) [reply src/components/Swap/Approval/index.tsx:13]: I see that it's convenient to use it this way, but it's hard to understand from the naming etc. why you have to access the first part of an array. Maybe we just remove the `tenantName` from the array and only return the data of the current tenant here? The `tenantName` probably is not required most of the time.

**Review by ebma — APPROVED** [review]
Added some comments but the code looks good overall 👍

File: src/components/Asset/Selector/Modal/index.tsx (hunk 1 above)
Comment (nejcm) [reply src/components/Asset/Selector/Modal/index.tsx:43]: So on nabla we have a constant with images from coingecko. Maybe there is a better way

File: src/components/Pools/Swap/columns.tsx (hunk 2 above)
Comment (nejcm) [reply src/components/Pools/Swap/columns.tsx:51]: This is something we did not resolve yet on nabla side either. We have a meeting tmr

File: src/components/Swap/Approval/index.tsx (hunk 3 above)
Comment (nejcm) [reply src/components/Swap/Approval/index.tsx:13]: Sure thing

File: src/components/Asset/Selector/Modal/index.tsx (hunk 1 above)
Comment (ebma) [reply src/components/Asset/Selector/Modal/index.tsx:43]: So you have a hard-coded mapping of token symbol to logo?

File: src/components/Swap/Approval/index.tsx (hunk 3 above)
Comment (nejcm) [reply src/components/Swap/Approval/index.tsx]: Updated

**Review by ebma — APPROVED** [review]
Looks good, thanks

File: src/components/Asset/Selector/Modal/index.tsx (hunk 1 above)
Comment (nejcm) [reply src/components/Asset/Selector/Modal/index.tsx:43]: No logo currently. So in that avatar if the image is missing it defaults to showing the symbol. This is useful even once we have images is one does not load or is missing.

**Comment by nejcm** [comment]
@ebma I dont have the perms to merge so you can do it.

**Pull request merged**